src/components/footer/EsFooter.vue

```diff
@@ -138,20 +146,33 @@
 
     watch: {
       $route(to) {
+        const getServiceTypeId = to.fullPath.split('nails/')[1];
+        const getGiftsId = to.fullPath.split('carduri-cadou/')[1];
         this.isProPage = to.fullPath.includes('easepro');
         this.isProWave = to.fullPath.includes('easepro/');
-        this.isClientWave = to.fullPath.includes('client/') || to.fullPath.includes('pro/');
+        this.isClientWave = to.fullPath.includes('client/') || 
```

Comment:
Instead of this, you can create an array of accepted links, such as:

const array = ['a', 'b', 'c'], and do: array.includes(to.fullPath)

Reply:
Good point! It will clean up a little bit the code and will be more maintainable. THX!